src/main/java/edu/byu/cs/autograder/Grader.java

```diff
@@ -124,7 +124,7 @@ private Rubric evaluateProject(RubricConfig rubricConfig) throws GradingExceptio
                     case PASSOFF_TESTS -> new PassoffTestGrader(gradingContext).runTests();
                     case UNIT_TESTS -> new UnitTestGrader(gradingContext).runTests();
                     case QUALITY -> new QualityGrader(gradingContext).runQualityChecks();
-                    case GITHUB_REPO -> new GitHubAssignmentGrader().grade();
+                    case GITHUB_REPO -> new GitHubAssignmentGrader().grade(commitVerificationResult);
```

Comment:
I like this division of responsibility where the "Phase Grader" is now responsible for verifying the state of the commit verification result.

Reply:
Thanks. That was @Fiwafoofa 's idea originally, I'm just readding his code